metadata-ingestion/README.md

```diff
@@ -483,13 +483,18 @@ Pull metadata from DBT output files:
 - [dbt catalog file](https://docs.getdbt.com/reference/artifacts/catalog-json)
   - This file contains schema data.
   - DBT does not record schema data for Ephemeral models, as such datahub will show Ephemeral models in the lineage, however there will be no associated schema for Ephemeral models
+  - Not required if platform configuration provided.
+- platform: Optional.
+  - If provided the ingestion tool will not attempt to load a catalog file.
+  - In this case we assume DBT metadata is enriching an existing data platform.
```

Comment:
In which case?

Reply:
improved the wording, marking as resolve.